src/main/java/org/opensrp/queue/ResourceEvaluatorMessage.java

```diff
@@ -0,0 +1,42 @@
+package org.opensrp.queue;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.ibm.fhir.model.resource.QuestionnaireResponse;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.ToString;
+import org.smartregister.domain.Action;
+import org.smartregister.pathevaluator.TriggerType;
+
+import java.io.Serializable;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
```

Comment:
for the To String you may want to implement custom so that only the Ids are shown for easy logging

Reply:
Done

Reply:
You then remove the ToString annotation

Reply:
Done